docs/src/components/DocSearch.css

```diff
@@ -1,3 +1,11 @@
+:root {
```

Comment:
@AlbertCarreras can we avoid `:root` and use a wrapping CSS class instead?

Reply:
DONE